GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -320,7 +320,9 @@ public void Clear()
             _backgroundScrollTo = 0;
 
             // Force the background thread to be killed, we need to be sure no background processes are running. Not the best practice, but safe.
-            _backgroundThread.Abort();
+            // TODO: not supported in .NET Core
+            //_backgroundThread.Abort();
+            _backgroundThread = null;
```

Comment:
@spdr870 could you please have a look.

[`Thread.Abort` is no longer supported](https://github.com/dotnet/coreclr/issues/20705), so we may need to re-write the handling.
This may be a possible solution https://stackoverflow.com/q/50957086/2338036

Reply:
This is an issue, because we need to make sure that no background thread is modifying the graph when we clear it. But I think I there is a simple solution. We need to move the (messy) code that creates RevisionGraph and starts the thread to a separate class. Also move the variable keeprunning to that class. Then in the gridview, in the clear function, set keeprunning to false in this new class and then create a new instance af that new class. Instead of killing the thread and resetting the state, it will let the thread stop in a safe way. The new instance will have it's own state, we do not have to worry about the state being modified by another thread. It's a better solution I think, because we do not need the threadabortexception.